service/history/replication/sequential_batch_queue.go

```diff
@@ -41,8 +41,9 @@ type (
 		taskQueue                    collection.Queue[*batchedTask]
 		lastTask                     *batchedTask
 		batchedIndividualTaskHandler func(task TrackableExecutableTask)
```

Comment:
this var is not used anywhere anymore?

Reply:
Yes. Will remove.

Reply:
Actually I decided to leave this var here to keep the interface consistent and as a reminder that the handler should be passed in. We are not using it now because we are taking a shortcut.